### ISSUE TITLE
Raspberry pi 3 + OpenELEC lights random led on power on and don't react

### ISSUE THREAD

**Issue by ajarzyn** [hyperion-project/hyperion#603]
Hi,

Issue is that I am able to light up some of the leds but it seems that it isn't working as it should.  At first I power on my RPi, then connect led to the power supply. What happens is that some of my leds lights up (but max 10 leds) and the rest is inactive. I've tried to change color, run test script but no change in lightening pattern same leds are active no change can be observe. Then I've tried to disconnect power supply from leds and connect it again, what I observed that sometime patterns change, and lights some of the other leds than previously.

tl:dr;
1. Not all of the leds are lightened
2. Led strip do not react to any controlling attempts.

1. RPi3 and OpenELEC
2. WS2812B 5m led strip
a) RPi3 (6th pin ground, 12th pin control)
b) power supply  Mean Well LRS-100-5 - 5V / 18A / 90W (5v through 1N4007 1A / 1000V)
c) connected only from one side of the strip

Hope you can help me solve the issue.
Regards

**Comment by penfold42** [comment]
Where's your config file ?

**Comment by ajarzyn** [comment]
I'll upload it in few hours. Apart of the path which i changed to "/storage/hyperion/effects" no other changes has been made. And the file was created by Hyperion tool.

Anyway pleas stay tuned I will upload it in around 10 hours.

Cheers

**Comment by ajarzyn** [comment]
Hi there this is my config file:

http://pastebin.com/7nQt9BAG

**Comment by tpmodding** [comment]
why did you commented the xbmc part out? :)

// 	"xbmcVideoChecker" : 
// 	{
// 		"xbmcAddress" : "127.0.0.1",
// 		"xbmcTcpPort" : 9090,
// 		"grabVideo" : true,
// 		"grabPictures" : true,
// 		"grabAudio" : true,
// 		"grabMenu" : false,
// 		"grabScreensaver" : true,
// 		"enable3DDetection" : true
// 	},

try this pls
http://pastebin.com/GgdYP57t

**Comment by ajarzyn** [comment]
Hi tpmodding,

I've tried your configuration file but nothing happened. Leds don't lights up at all now.
I've reistaleed hyperion but this haven't solve the issue.

My wired setup is just as on the picture:
https://cloud.githubusercontent.com/assets/129956/6201128/8d039fb0-b498-11e4-9500-7b68430a2784.png

**Comment by tpmodding** [comment]
could you post your output log when you start hyperion per command?

**Comment by ajarzyn** [comment]
Sorry for my question but where I can find how to perform that action? I can't find that information anywhere.

Sorry for bothering you :/

**Comment by brindosch** [comment]
@tpmodding 
time for a hypercon update :)

**Comment by ajarzyn** [comment]
@brindosch what do you mean?

**Comment by tpmodding** [comment]
@ajarzyn not relevant for you :)

please use this one:

/storage/hyperion/bin/hyperiond.sh /storage/.config/hyperion.config.json

then you will get an output :)

**Comment by ajarzyn** [comment]
So as I can see Kodi connection problem:

http://pastebin.com/5ngfR9FS

**Comment by brindosch** [comment]
Please check FAQ
https://github.com/hyperion-project/hyperion/wiki/FAQ-Frequently-Asked-(Problems)-Questions

**Comment by ajarzyn** [comment]
Yeah sorry about that I've activated Kodi control, though I did it before or it could just not save.

Anyway here you have my log after I correct my mistake:

http://pastebin.com/H07gFB5S

(Still leds are idle :/ )

**Comment by ajarzyn** [comment]
Hi there,

Do you guys have some other suggestions maybie?

Thanks in advance.

**Comment by tpmodding** [comment]
can you make an picture where i can see the connected pins from rpi and the wires to your led please?

**Comment by ajarzyn** [comment]
This is it:
http://imgur.com/a/iEj8O

In case of questions here's some description:
http://imgur.com/7G872Dc

I hope this is clear.

And thank you that you haven't give up on me ;)

**Comment by tpmodding** [comment]
pictures are very bad... cant count the pins on rpi etc... :D 

can you do me favor, and cut the shrink tubing...the chinese often change the colors... :)

**Comment by ajarzyn** [comment]
Are this photos better:
http://imgur.com/a/xzoLX

Pic 1: Shrink tubing removed

On the second picture there are 3 cables connected 1. Ground on RPi (6th PIN) 2. V- 3. GND from led strip. Black and pink are not connected at all they just hung in there ;).
You can also observe red cable which leads to 5V on lead strip connected through diode with V+ (black/white cable).

3rd picture. Yellow (V-/GND) connected to 6th PIN (3rd from right or top as you look)
Green cable (controller?) connected to 12th PIN (6th ....)

**Comment by brindosch** [comment]
Is this a resistor at the 5V wire?. Are you sure these gpio connectors are "connected" pin layout looks good.

**Comment by ajarzyn** [comment]
I've removed diod, and tried to run some test, and set specific color by hypercon. Leds do not respond. 

I don't know what happened but they don't lights up at all anymore.

As I run color set though hypercon, there are logs in the console, but nothing really change led status.:(

**Comment by brindosch** [comment]
Hyperion is running? could you post a example output? (The log button won´t work on OpenELEC atm, will be fixed with next release)

**Comment by brindosch** [comment]
Hyperion have a new home!
Please read the announcement post and the changelog for V1.02.0
www.hyperion-project.org

Please have a look at our new wiki! (EN/DE)
wiki.hyperion-project.org

All discussions/questions/development threads should be moved to the new forum!
We will close this issue now
